integration-tests/src/test/java/com/redhat/service/bridge/integration/tests/resources/IngressResource.java

```diff
@@ -0,0 +1,44 @@
+package com.redhat.service.bridge.integration.tests.resources;
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+
+import io.restassured.http.ContentType;
+import io.restassured.http.Headers;
+import io.restassured.response.Response;
+
+public class IngressResource {
+
+    public static Response optionsJsonEmptyEventResponse(String token, String endpoint) {
+        ByteArrayInputStream cloudEventStream = new ByteArrayInputStream("".getBytes(StandardCharsets.UTF_8));
+        return ResourceUtils.jsonRequest(token)
+                .body(cloudEventStream)
+                .contentType(ContentType.JSON)
+                .options(endpoint + "/events");
+    }
+
+    public static Response postJsonEventResponse(String token, String endpoint, InputStream cloudEventStream, Headers headers) {
+        return ResourceUtils.jsonRequest(token)
+                .headers(headers)
+                .body(cloudEventStream)
+                .contentType(ContentType.JSON)
+                .post(endpoint + "/events");
+    }
+
+    public static Response postJsonEventToFullURLResponse(String token, String url, InputStream cloudEventStream, Headers headers) {
```

Comment:
Is this needed? I think the functionality can be achieved by using `postJsonEventResponse` with some non existing endpoint parameter.